### PR TITLE
{bio}[foss/2023a] hdWGCNA v0.3.00 w/ R 4.3.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3136.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/49a5836483c01b834c7e80eef8e49875 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/h/hdWGCNA/hdWGCNA-0.3.00-foss-2023a-R-4.3.2.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+easyblock = 'Bundle'
+
+name = 'hdWGCNA'
+version = '0.3.00'
+versionsuffix = '-R-%(rver)s'
+local_biocver = '3.16'
+
+homepage = 'https://github.com/vibscc/DALI'
+description = "R-package for the analysis of single-cell TCR/BCR data in the Seurat ecosystem"
```
Comment (branfosj) [inline easybuild/easyconfigs/h/hdWGCNA/hdWGCNA-0.3.00-foss-2023a-R-4.3.2.eb:9]:
```suggestion
homepage = 'https://smorabit.github.io/hdWGCNA/'
description = """hdWGCNA is an R package for performing weighted gene co-expression network analysis (WGCNA) in high 
 dimensional transcriptomics data such as single-cell RNA-seq or spatial transcriptomics. hdWGCNA is highly modular 
 and can construct context-specific co-expression networks across cellular and spatial hierarchies. hdWGNCA identifies 
 modules of highly co-expressed genes and provides context for these modules via statistical testing and biological 
 knowledge sources. hdWGCNA uses datasets formatted as Seurat objects."""
```

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/35db06016fb2e413661bc2cc30c5b6b8 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**